tests/drivers/build_all/dts.fixup

```diff
@@ -0,0 +1,31 @@
+#if defined(CONFIG_HAS_DTS_I2C_DEVICE)
+
+#ifndef CONFIG_FXAS21002_NAME
+#define CONFIG_FXAS21002_NAME ""
+#define CONFIG_FXAS21002_I2C_ADDRESS 0
+#define CONFIG_FXAS21002_I2C_NAME ""
+#endif
+
+#ifndef CONFIG_FXOS8700_NAME
+#define CONFIG_FXOS8700_NAME ""
+#define CONFIG_FXOS8700_I2C_NAME ""
+#define CONFIG_FXOS8700_I2C_ADDRESS 0
+#endif
+
+#ifndef CONFIG_HTS221_NAME
+#define CONFIG_HTS221_NAME ""
+#define CONFIG_HTS221_I2C_MASTER_DEV_NAME ""
+#endif
+
+#ifndef CONFIG_LIS3MDL_NAME
+#define CONFIG_LIS3MDL_NAME ""
+#define CONFIG_LIS3MDL_I2C_MASTER_DEV_NAME ""
+#define CONFIG_LIS3MDL_I2C_ADDR 0x1e
```

Comment:
Is there a reason you used 0xe here but 0 elsewhere?

Reply:
I guess this is because _NAME and _DEV_NAME are used as char * and accept null string while _I2C_ADDR stands for a u16_t and expects a value

Reply:
Also, the LIS3MDL driver has a change that the LSB or something like that is set.  Otherwise I'd use 0.

Reply:
Here's why I picked 0x1e (taken from disco dts file):

in drivers/sensor/lis3mdl/lis3mdl.h

```
/* guard against invalid CONFIG_I2C_ADDR values */
#if (CONFIG_LIS3MDL_I2C_ADDR & LIS3MDL_I2C_ADDR_MASK) != LIS3MDL_I2C_ADDR_BASE
#error "Invalid value for CONFIG_LIS3MDL_I2C_ADDR"
#endif
```